services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -327,7 +327,13 @@ async def task_ports_inputs_pull(
     port_keys: list[str] | None,
     mounted_volumes: MountedVolumes,
     app: FastAPI,
+    *,
+    inputs_pulling_enabled: bool,
 ) -> int:
+    if not inputs_pulling_enabled:
+        _logger.info("Received request to pull inputs but was ignored")
+        return 0
```

Comment:
what means return 0 in this context??

Reply:
It means no bytes were transferred since, which makes sense.